build_tools/bazel/iree_check_test.bzl

```diff
@@ -176,8 +178,9 @@ def iree_check_test_suite(
     Args:
       name: name of the generated test suite.
       srcs: source mlir files containing the module.
-      target_backends_and_drivers: backend/driver pairs to compile and run the
-          module, respectively.
+      target_backends_and_drivers: list of ("backend", "driver") tuples to
+          compile and run the module, respectively. If "driver" is omitted then
+          only compilation will be tested.
```

Comment:
I'm not sure if this is actually true on our CI... https://github.com/openxla/iree/blob/main/build_tools/bazel/build_core.sh only runs a wildcard `bazel test`, which won't build any non-test targets?